pkg/garden/ted/glob.hoon

```diff
@@ -9,7 +9,7 @@
 |^
 ?-  -.loc
   %http  (fetch-http url.loc)
-  %ames  (fetch-ames ship.loc base)
+  %ames  !!  ::NOTE  done within docket itself
```

Comment:
should we add a reminder to move this logic back here when the desk routing in gall is done?

Reply:
Sure.  If we later fetch globs file by file, then doing that in a thread would probably be better.